test/integration/middleware_test.rb

```diff
@@ -31,7 +31,6 @@ class MiddlewareIntegrationTest < ActionDispatch::IntegrationTest
     test 'it is added Content-Security-Policy on welcome pages' do
       Domain.stubs(:any?).returns(false)
       visit '/domains'
```

Comment:
This should probably also be changed to
```suggestion
      visit '/domains/help'
```